A59-lrs-named-metrics.md

```diff
@@ -0,0 +1,97 @@
+A59: xDS LRS Named Metrics Support
```

Comment:
A59 through A63 are already taken by other pending gRFCs.  This will be A64.

Please update both this title and the filename accordingly.

Reply:
Done.